lib/linter-cpplint.coffee

```diff
@@ -1,63 +1,72 @@
 {CompositeDisposable} = require 'atom'
-linterPath = atom.packages.getLoadedPackage("linter").path
-Linter = require "#{linterPath}/lib/linter"
-path = require 'path'
 
-class LinterCpplint extends Linter
-  # The syntax that the linter handles. May be a string or
-  # list/tuple of strings. Names should be all lowercase.
-  @syntax: ['source.cpp']
-
-  linterName: 'cpplint'
-
-  # A regex pattern used to extract information from the executable's output.
-  regex: '.+:(?<line>\\d+):\\s+(?<message>.*).+\\[\\d\\]$'
-  regexFlags: 'm'
-
-  defaultLevel: 'warning'
-
-  isNodeExecutable: no
-
-  errorStream: 'stderr'
-
-  constructor: (editor) ->
-    super(editor)
+class LinterCpplint
```

Comment:
It would be nice if you could merge this class back in the main as it doesn't contain anything complicated